logging/src/main/java/life/qbic/logging/api/Logger.java

```diff
@@ -0,0 +1,44 @@
+package life.qbic.logging.api;
+
+/**
+ * Simplistic interface of a {@link Logger}, that enables basic logging functionality for different
+ * log levels.
+ *
+ * @since 1.0.0
+ */
+public interface Logger {
+
+  /**
+   * Logs a message on DEBUG level
+   *
+   * @param message the message content
+   * @since 1.0.0
+   */
+  void debug(String message);
+
+  /**
+   * Logs a message on ERROR level
+   *
+   * @param message the message content
+   * @since 1.0.0
+   */
+  void error(String message);
+
+  /**
+   * Logs a message on ERROR level with a {@link Throwable} provided
+   *
+   * @param message the message content
+   * @param t       the throwable to report
+   * @since 1.0.0
+   */
+  void error(String message, Throwable t);
```

Comment:
What is the throwable meant to be here? Is it the cause for the error, is it something else? do we want to specify that further?

Reply:
I will rename it to `cause`, to make it more explicit.